modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowWebContentArticle.testcase

```diff
@@ -28,80 +28,6 @@ definition {
 		}
 	}
 
-	@description = "Verify that the user can modify the Web Content's workflow"
```

Comment:
Removing test as this is already tested on WorkflowBlogs#CanHaveDifferentBlogsWithDifferentWorkflows.